README.md

```diff
@@ -74,10 +74,12 @@ or a [Jupyter notebook](https://docs.jupyter.org/en/latest/running.html), and tr
 ``` python
 import pygmt
 fig = pygmt.Figure()
-fig.coast(projection="H10c", region="g", frame=True, land="gray")
+fig.coast(projection="N10c", region="g", frame=True, land="tan", water="lightblue")
+fig.text(position="MC", text="PyGMT", font="60p,Helvetica-Bold,red@75")
 fig.show()
 ```
 
+You should see a global map with land and water masses colored in tan and lightblue, respectively, with a semi-transparent text "PyGMT" on top. 
```

Comment:
```suggestion
You should see a global map with land and water masses colored in tan and lightblue, respectively, with the semi-transparent text "PyGMT" on top. 
```
To make it consistent with the text in install.md.

Reply:
See commit https://github.com/GenericMappingTools/pygmt/pull/3249/commits/869be37d18481ccf379df5268fb561753574e73d.